features/karaf/openhab-core/src/main/feature/feature.xml

```diff
@@ -215,6 +215,13 @@
     <bundle>mvn:org.openhab.core.bundles/org.openhab.core.config.discovery.usbserial.linuxsysfs/${project.version}</bundle>
   </feature>
 
+  <feature name="openhab-core-io-bin2json" version="${project.version}">
```

Comment:
I see your commit fb4604fb832c188074235ba5d022e62947a53d54
Why did you remove it from the base runtime again?
I don't know your discussion with Kai. Should it be an optional feature that is installed on request or should it part of every openHAB installation?